### PR TITLE
E-mail address is not allowed in "display name"

### PR DESCRIPTION
**Pull request by jeroendesloovere**
Because this can cause errors when trying to send mails with an e-mail as a name.

### CODE REVIEW

File: src/Backend/Modules/Profiles/Installer/Data/locale.xml [hunk 1]
```diff
@@ -649,6 +649,18 @@
...
         <translation language="el"><![CDATA[Αυτή η διεύθυνση email είναι σε χρήση.]]></translation>
       </item>
+      <item type="error" name="EmailNotAllowed">
+        <translation language="nl"><![CDATA[E-mailadres is hier niet toegestaan.]]></translation>
+        <translation language="en"><![CDATA[E-mailaddress not allowed here.]]></translation>
+        <translation language="it"><![CDATA[L'indirizzo email non permesso qui.]]></translation>
+        <translation language="ru"><![CDATA[Адрес электронной почты здесь не допускается.]]></translation>
+        <translation language="zh"><![CDATA[电子邮件地址这里不允许]]></translation>
+        <translation language="de"><![CDATA[E-Mail-Adresse hier nicht erlaubt.]]></translation>
+        <translation language="fr"><![CDATA[Adresse e-mail non autorisé ici.]]></translation>
```
Comment (WouterSioen) [inline src/Backend/Modules/Profiles/Installer/Data/locale.xml]: I don't think this is correct french ;)

Comment (jeroendesloovere) [inline src/Backend/Modules/Profiles/Installer/Data/locale.xml]: Fuck google translate then!

File: src/Backend/Modules/Profiles/Installer/Data/locale.xml [hunk 2]
```diff
@@ -649,6 +649,18 @@
...
+        <translation language="nl"><![CDATA[E-mailadres is hier niet toegestaan.]]></translation>
+        <translation language="en"><![CDATA[E-mailaddress not allowed here.]]></translation>
+        <translation language="it"><![CDATA[L'indirizzo email non permesso qui.]]></translation>
+        <translation language="ru"><![CDATA[Адрес электронной почты здесь не допускается.]]></translation>
+        <translation language="zh"><![CDATA[电子邮件地址这里不允许]]></translation>
+        <translation language="de"><![CDATA[E-Mail-Adresse hier nicht erlaubt.]]></translation>
+        <translation language="fr"><![CDATA[Adresse e-mail est pas autorisée ici.]]></translation>
+        <translation language="hu"><![CDATA[E-mail cím nem engedélyezett.]]></translation>
+        <translation language="es"><![CDATA[Dirección de correo electrónico no permitida aquí.]]></translation>
+        <translation language="el"><![CDATA[διεύθυνση ηλεκτρονικού ταχυδρομείου δεν επιτρέπεται εδώ.]]></translation>
```
Comment (WouterSioen) [inline src/Backend/Modules/Profiles/Installer/Data/locale.xml]: Do you speak all these languages? Isn't it safer to leave them blank?

**Comment by WouterSioen** [comment]
Why would this cause errors? I don't really see the issue?

File: src/Backend/Modules/Profiles/Installer/Data/locale.xml [hunk 3]
```diff
@@ -656,7 +656,7 @@
         <translation language="ru"><![CDATA[Адрес электронной почты здесь не допускается.]]></translation>
         <translation language="zh"><![CDATA[电子邮件地址这里不允许]]></translation>
         <translation language="de"><![CDATA[E-Mail-Adresse hier nicht erlaubt.]]></translation>
-        <translation language="fr"><![CDATA[Adresse e-mail non autorisé ici.]]></translation>
+        <translation language="fr"><![CDATA[Adresse e-mail est pas autorisée ici.]]></translation>
```
Comment (WouterSioen) [inline src/Backend/Modules/Profiles/Installer/Data/locale.xml]: this should probably be `n'est pas autorisée`

**Comment by jeroendesloovere** [comment]
SpoonMail throws errors at us in an older fork cms website.

**Comment by WouterSioen** [comment]
This version of Fork does not use SpoonMail anymore. I thus see no reason to merge this?

**Comment by jeroendesloovere** [comment]
@WouterSioen So you love the fact that users fill in an e-mailadres as "display name"?
How unlogical.

**Comment by WouterSioen** [comment]
I don't love it, but I have no problem with it. If people want their email address to be displayed everywhere, it's their responsibility, not ours.

**Comment by jeroendesloovere** [comment]
Would be better if an email is not allowed in a display name.
Since sending emails to "jeroen@siesqo.be <jeroen@siesqo.be>" is rather spammy.

But if SwiftMailer supports this, then maby we should close this PR

**Pull request closed without merging**